### PR TITLE
Add UUID support for password reset

### PR DESCRIPTION
**Pull request by yarbelk**
Fix for #994 
Basically add some tests for the uuid field serialization in the urls, and force casting to string on the user_pk_to_url_str method

### CODE REVIEW

**Comment by yarbelk** [comment]
fixing python 2.6 ignoring of tests now (UUIDFields are not available in python2.6, so the one direction of test doesn't matter.)

**Comment by JshWright** [comment]
Django 1.6 (and therefore Python 2.6) are no longer supported. Why not drop it entirely from the Travis config?

**Comment by JshWright** [comment]
Since it seems kind of dodgy to hack around unsupported Django/Python versions in the test suite, I've opened PR1001

**Comment by yarbelk** [comment]
it doubled the joy :)

going to squash them all down.

File: allauth/account/utils.py [hunk 1]
```diff
@@ -362,16 +363,25 @@ def passthrough_next_redirect_url(request, url, redirect_field_name):
...
+    """
+    User = get_user_model()
+    if hasattr(models, 'UUIDField') and issubclass(type(User._meta.pk), models.UUIDField):
+        return user.pk.hex
+
     ret = user.pk
     if isinstance(ret, six.integer_types):
         ret = int_to_base36(user.pk)
-    return ret
+    return str(ret)
```
Comment (pennersr) [inline allauth/account/utils.py]: This str() is superfluous right?

Comment (yarbelk) [inline allauth/account/utils.py]: yes.  from a squashed merge before i realized that I had to return hex.  will fix it.

**Pull request merged**